wanshi/visualizations/multi_cohort_scatter/multi_cohort_scatter.py

```diff
@@ -0,0 +1,363 @@
+# !/usr/bin/env python3
+
+__author__ = "Marco Gustav"
+__copyright__ = "Copyright 2023, Kather Lab"
+__license__ = "MIT"
+__version__ = "0.1.0"
+__maintainer__ = ["Marco Gustav", "Jeff"]
+__email__ = "marco.gustav@tu-dresden.de"
+
+import argparse
+import os
+from pathlib import Path
+from typing import List
+
+import matplotlib.patches as ptch
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from sklearn import metrics
+
+# pd float shown with 2 decimal places
+pd.set_option("display.float_format", lambda x: "%.5f" % x)
+
+
+def plot_bubble(
+    data_path_internal: str,
+    data_paths_external: List[str],
+    title: str,
+    outpath: str,
+    internal_cohort_name: str,
+    color_scheme: list,
+    format: str = "norm",
+    zoom_x_lower_thresh: float = 0.75,
```

Comment:
Those are really nice graphs generated, some of the settings could be saved as template for future use.